src/components/Lend/LoanSearch/LoanSearchInterface.vue

```diff
@@ -222,12 +225,41 @@ export default {
 		};
 	},
 	apollo: {
-		query: userIdQuery,
-		preFetch: true,
-		result({ data }) {
-			this.userId = data?.my?.userAccount?.id ?? null;
+		preFetch(config, client) {
+			return client.query({
+				query: userIdQuery
+			}).then(() => {
+				return client.query({
+					query: itemsInBasketQuery
+				});
+			});
 		},
 	},
+	created() {
+		const basketId = this.cookieStore.get('kvbskt');
```

Comment:
I know we use this everywhere, but `kvbskt` seems like a good constant to share.

Reply:
Some type of global, but also update access could be useful and worth exploring more. Def an improvement I'd like to see in the CMS page server. Ultimately, this data exists in it's most up to date form in the cookie does reside in the cookie and at this point the ability to access on both the server and client this way works pretty well.

Reply:
That's true. I also just meant the string `kvbskt` itself in code in a global constant.